packages/cli/fern-definition/schema/src/utils/visitRawTypeReference.ts

```diff
@@ -104,7 +104,9 @@ export function visitRawTypeReference<R>({
         case RawPrimitiveType.boolean:
             return visitor.primitive({
                 v1: PrimitiveTypeV1.Boolean,
-                v2: undefined
+                v2: PrimitiveTypeV2.boolean({
+                    default: _default != null ? (_default as boolean) : undefined
+                })
```

Comment:
@eyw520 do we need to add these v2 constructors to all the `undefined` values above?

Reply:
I've added it for all the ones that currently support them, but there seems to be gaps in the primitiveTypeV2 definitions. For instance, we support defaults for `long` values, but not `float` values.